product/product.css

```diff
@@ -0,0 +1,221 @@
+    border: 0px solid var(--BACKGROUND-COLOR);
+    height: 1.125rem;
+    width: 1.125rem;
+    border-radius: 1.563rem;
+    background: var(--PRIMARY-COLOR);
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin-top: -7px;
+}
+
+
+/* Category */
+
+/* Category List */
+
+ul li {
+    font-weight: 600;
+    font-size: 14px;
+}
+
+/* Input Category ( checkbox ) */
+input[type=checkbox] {
+    position: relative;
+    cursor: pointer;
+    right: 10px;
+}
+
+/* Display for checkbox */
+input[type=checkbox]:before {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 1.25rem;
+    height: 1.25rem;
+    top: 0;
+    left: 0;
+    background-color: #e9e9e9;
+}
+
+
+input[type=checkbox]:checked:before {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 1.25rem;
+    height: 1.25rem;
+    top: 0;
+    left: 0;
+    background-color: var(--PRIMARY-COLOR);
+}
+
+input[type=checkbox]:checked:after {
+    content: "";
+    display: block;
+    width: 5px;
+    height: 10px;
+    border: solid white;
+    border-width: 0 2px 2px 0;
+    -webkit-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+    position: absolute;
+    top: 2px;
+    left: 6px;
+}
+
+/* Input Rating(Radio) */
+input[type=radio] {
+    position: relative;
+    cursor: pointer;
+    right: 10px;
+}
+
+/* Display for Radio */
+input[type=radio]:before {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 1.25rem;
+    height: 1.25rem;
+    border-radius: 50%;
+    top: -1px;
+    left: -1px;
+    background-color: #e9e9e9;
+}
+
+
+input[type=radio]:checked:before {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 1.25rem;
+    height: 1.25rem;
+    border-radius: 50%;
+    top: -1px;
+    left: -1px;
+    background-color: var(--PRIMARY-COLOR);
+}
+
+input[type=radio]:checked:after {
+    content: "";
+    display: block;
+    border: 2px solid white;
+    border-radius: 50%;
+    position: absolute;
+    margin: 2px;
+    padding: 5px;
+}
+
+
+/* Products */
+main {
+    margin-left: 20%;
+}
+
+main h2 {
+    font-size: 1.5rem;
+    margin-top: -5px;
+    padding-left: 2.5rem;
+    align-self: center;
+}
+
+.product-container{
+    margin-top: 1.5rem;
+    display: grid;
+    grid-template-columns:repeat(auto-fit, 18rem);
+    grid-auto-rows: auto;
+    grid-gap: 1rem;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.m-head {
+    margin: 1.5rem 0rem 1rem;
+}
+
+.highlight-text {
+    color: var(--SECONDARY-COLOR);
+    font-size: 0.8rem;
+    text-transform: capitalize;
+}
+
+.card-badge:hover{
+      cursor: pointer;
+  }
+
+  .card-header .category{
+      color:var(--TEXT-COLOR);
+      position: absolute;
+      bottom: 1.5rem;
+      left: 1rem;
+      background-color:var(--SECONDARY-COLOR);
+      padding: 3px;
+      border-radius:0px 5px 5px 0px;
+      font-size: 14px;
```

Comment:
Use rem

Reply:
we can use px for less than 1rem i.e 16px, right?

Reply:
Ohk you haven't done 62.5% thing got it.